### PR TITLE
Use CMS_CACHE_DURATIONS, when configured

### PR DESCRIPTION
**Pull request by mkoistinen**
Should address #4931.

### CODE REVIEW

File: cms/utils/conf.py [hunk 1]
```diff
@@ -67,9 +66,13 @@ def wrapper():
...
-    return {
+    """
+    Returns the setting: CMS_CACHE_DURATIONS or will attempt to build it from
+    the deprecated settings or will return the defaults.
+    """
+    return getattr(settings, 'CMS_CACHE_DURATIONS', {
         'menus': getattr(settings, 'MENU_CACHE_DURATION', 60 * 60),
-        'content': get_cms_setting('CONTENT_CACHE_DURATION'),
+        'content': getattr(settings, 'CMS_CONTENT_CACHE_DURATION', 60),
         'permissions': 60 * 60,
```
Comment (yakky) [inline cms/utils/conf.py]: This should be fixed too, right?

**Comment by yakky** [comment]
Shouldn't this be something like:

    return getattr(settings, 'CMS_CACHE_DURATIONS', {
        'menus': 60 * 60,
        'content': 60,
        'permissions': 60 * 60,
    }

Individual settings are no longer available, and we should just drop them

**Comment by mkoistinen** [comment]
I was just adding a layer of convenience for migrating older projects... but, I'm not passionate about it.

**Comment by yakky** [comment]
setting changed in 2.3 or 2.4 (IIRC) so I guess we can just drop the "compatible" setting

File: cms/utils/conf.py [hunk 2]
```diff
@@ -67,9 +66,13 @@ def wrapper():
...
 def get_cache_durations():
-    return {
+    """
+    Returns the setting: CMS_CACHE_DURATIONS or will attempt to build it from
+    the deprecated settings or will return the defaults.
+    """
+    return getattr(settings, 'CMS_CACHE_DURATIONS', {
         'menus': getattr(settings, 'MENU_CACHE_DURATION', 60 * 60),
-        'content': get_cms_setting('CONTENT_CACHE_DURATION'),
+        'content': getattr(settings, 'CMS_CONTENT_CACHE_DURATION', 60),
```
Comment (czpython) [inline cms/utils/conf.py]: why not use ''get_cms_setting''?

**Comment by czpython** [comment]
:+1:

File: cms/utils/conf.py (hunk 2 above)
Comment (mkoistinen) [inline cms/utils/conf.py]: Because that setting is no longer supported. Doesn't matter though, I've removed it altogether based on @yakky's note.

**Comment by yakky** [comment]
Linking to #4960 to update the changelog

**Comment by yakky** [comment]
@mkoistinen I see that tests fail: do we want this in 3.1.5/3.2.1?

**Pull request closed without merging**